src/components/Body.js

```diff
@@ -6,6 +6,13 @@ const Body = props => {
     <div className="bg-light">
       <div className="body">
         <p className="tx-dark">{props.text}</p>
+        <p className="tx-dark">These links may be helpful:</p>
```

Comment:
Lets make this an H2 tag or something similar:
```suggestion
        <h2 className="tx-dark">These links may be helpful:</h2>
```